vulture/core.py

```diff
@@ -184,13 +184,18 @@ def exclude(name):
             else:
                 self.scan(module_string, filename=module)
 
-            whitelist_names = ['stdlib.py']
-            for name in whitelist_names:
+            for name in self.defined_imports:
                 path = os.path.join('whitelists', name)
+                path += '.py'
```

Comment:
please put this on the previous line: `path = os.path.join('whitelists', name) + '.py'`